dask_expr/expr.py

```diff
@@ -983,6 +986,12 @@ def _simplify_up(self, parent):
             )
 
 
+class Fillna(Blockwise):
```

Comment:
Blockwise is correct, but Elemwise is more correct (this operation operates block by block, but also row-by-row).  This is useful for some optimizations.  For example `Len(Fillna(x)) -> Len(x)`

```suggestion
class Fillna(Elemwise):
```